php/antidot/composer.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "antidot-fw/reactive-starter",
+  "description": "Reactive Antidot Framework Apllication Starter",
+  "type": "project",
+  "license": "BSD-2-Clause",
+  "authors": [
+    {
+      "name": "kpicaza"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/antidot-framework/psr11-monolog.git"
+    }
+  ],
+  "require": {
+    "php": "^7.4|^8.0",
+    "ext-json": "*",
+    "antidot-fw/react-framework": "^1.0.3",
+    "antidot-fw/fast-router-adapter": "^0.1.0",
+    "antidot-fw/cli": "^1.1.0",
+    "antidot-fw/container": "^0.1.0",
+    "antidot-fw/dev-tools": "^0.1.2",
+    "antidot-fw/event-dispatcher": "^2.0.3",
+    "antidot-fw/framework": "^0.1.1",
+    "antidot-fw/logger": "^1.1.0",
+    "antidot-fw/symfony-config-translator": "^1.1.0",
+    "antidot-fw/yaml-config-provider": "^0.1.0",
+    "webmozart/assert": "^1.7.0",
+    "wshafer/psr11-monolog": "^3.0.0|@dev"
```

Comment:
do we need to specify each version ?
I mean is their not a main package, like `antidot-fw/framework` which require all components ?

Reply:
also I prefer using tilde ranges, to ensure that version is in-sync with what we show in `README.md`

Reply:
Is the standard installation, which adds some functionality like event dispatcher, logger, cli, dev-tools... I will update the PR accordingly using only the minimal installation, thanks for the advice;- D